### PR TITLE
Github #278: change lastInsertId to return the last sequence number

### PR DESCRIPTION
**Pull request by v-kaywon**
Fix for #278.
Change lastInsertId to return the last sequence number when the sequence name is provided. Previously when a table name is provided, the driver returns the last id inserted of that table. Now when a table name is given, an empty string is returned.

### CODE REVIEW

**Comment by v-kaywon** [comment]
@meet-bhagdev: this PR does not include the connection option, it simply change the lastInsertId function to return the last sequence number. Depending on the decision if a connection option for sequence number is wanted, the code will/will not be changed.

**Comment by v-kaywon** [comment]
@yitam :  did that already :)

**Comment by yitam** [comment]
So when the user is running an old sql server but supplies a name to lastInsertId() the method simply returns an empty string?

**Comment by v-kaywon** [comment]
yes, in fast when the user tries to create a sequence, the stmt returned would be false

File: source/pdo_sqlsrv/pdo_dbh.cpp [hunk 1]
```diff
@@ -30,7 +30,7 @@ namespace {
 
 const char LAST_INSERT_ID_QUERY[] = "SELECT @@IDENTITY;";
 const size_t LAST_INSERT_ID_BUFF_LEN = 10;    // size of the buffer to hold the string value of the last insert id integer
-const char TABLE_LAST_INSERT_ID_QUERY[] = "SELECT IDENT_CURRENT(%s)";
+const char TABLE_LAST_INSERT_ID_QUERY[] = "SELECT CURRENT_VALUE FROM SYS.SEQUENCES WHERE NAME=%s";
```
Comment (Hadis-Fard) [inline source/pdo_sqlsrv/pdo_dbh.cpp]: it's better to change the `TABLE_LAST_INSERT_ID_QUERY` to Sequence.

**Comment by Hadis-Fard** [comment]
@david-puglielli can we close this PR now?

**Comment by david-puglielli** [comment]
Yes this PR can be closed.

**Pull request closed without merging**